qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -201,70 +202,98 @@ class WebEngineCaret(browsertab.AbstractCaret):
 
     @pyqtSlot(usertypes.KeyMode)
     def _on_mode_entered(self, mode):
-        pass
+        if mode != usertypes.KeyMode.caret:
+            return
+
+        self._tab.run_js_async(
+            javascript.assemble('caret', 'setInitialCursor',
+                                platform.platform()))
```

Comment:
`platform.platform()` is intended to be human-readable - you might want to pass `sys.platform` instead. See [this code](https://github.com/qutebrowser/qutebrowser/blob/v1.1.0/qutebrowser/utils/utils.py#L57-L60) for reference.